### PR TITLE
Is that good to begin with?

### PR DESCRIPTION
**Pull request by Chr1st1anSzech**
I don't know how to determine the actual project folder. Therefore I defined it hard-code. Environment.CurrentDirectory doesn't work in 1.3 So you have to change the path manually.

### CODE REVIEW

File: VaderSharp/VaderSharp/ConfigStore.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+namespace VaderSharp
+{
+    public class ConfigStore
+    {
+
+        private static ConfigStore config;
+
+        private Dictionary<string, double> boosterDict;
+        public Dictionary<string, double> BoosterDict { get { return boosterDict; } }
```
Comment (codingupastorm) [inline VaderSharp/VaderSharp/ConfigStore.cs:14]: Can we do get; private set; here? And other fields in ConfigStore

**Review by codingupastorm — CHANGES_REQUESTED** [review]
Looks good. Do you want to initialise and actually use the ConfigStore class in this pull req? Or I can put it in later, up to you

**Comment by Chr1st1anSzech** [comment]
Thanks. I'll make a couple of modifications. First of all I wanted to receive a first feedback. Is XML a good choice? Originally, I prefered JSON, however, there's no built-in JSON parser. You would have to use some external framework, e. g. Newtonsoft Json.NET.

**Comment by codingupastorm** [comment]
I definitely prefer JSON and would be happy to use Json.Net, but you have already put a heap of work in here (thanks!) and I don't think it will have much difference, so we can leave it.

Did you use some code to get the xml file? Even for only some parts of it? It could be useful to include it in a different directory at the top level (utils or scripts or so). I don't think the algorithm or word scores will change, but if they are updated in, say, the original vader in python it would be good to know if there is any useful code to port the new lists across.

**Comment by Chr1st1anSzech** [comment]
Yeah, JSON has a lesser overhead and I'm sure that it could be parsed faster. I could still change it if it's okay to use such a Framework.

I created the XML file by Hand. Nevertheless it's a good idea to add some methods to add (/remove) words to the list. Or to create the list from a normal (comma separated) text file.

**Comment by codingupastorm** [comment]
Hey @Chr1st1anSzech you put a heap of work into this so I'm going to merge it and make some changes 🔥 🔥 . Worst case can always keep it as an alternate means of loading the data.

**Pull request merged**